src/test/isolation2/sql/remove_temp_schemas.sql

```diff
@@ -0,0 +1,22 @@
+DO $$
```

Comment:
How about to use same single file for both regress and isolation2 tests to avoid duplication? You may use relative path with double points.

Reply:
I think about it, but in my opinion it is not good idea: these tests are different with different styles. Isolation2 tests  combine lines of one command into one line, regressions tests don't do it. Using relative path is bad idea, in the future it will lead to problems, if some directory is moved to the other path.

Reply:
After all, you accepted the suggestion!